src/Bicep.Core/Analyzers/Linter/LinterAnalyzer.cs

```diff
@@ -25,17 +25,24 @@ internal class LinterAnalyzer : IBicepAnalyzer
 
         private ConfigHelper configHelper = new ConfigHelper();
         private ImmutableArray<IBicepAnalyzerRule> RuleSet;
+        private ImmutableArray<IBicepAnalyzerDiagnostic> RuleCreationErrors;
+
+        internal const string FailedRuleCode = "Linter Rule Error";
```

Comment:
this won't be a config code, but should we still style the code like the other rule codes with lower case and hyphens?

Reply:
```suggestion
        internal const string FailedRuleCode = "linter-internal-error";
```

Reply:
For later, we might consider making it into a bicep-coded error.